cloudinit/util.py

```diff
@@ -78,6 +78,10 @@
 TRUE_STRINGS = ("true", "1", "on", "yes")
 FALSE_STRINGS = ("off", "0", "no", "false")
 
+# logrotate
```

Comment:
Being log-rotation implemented as package configuration, I think these hard-coded values are not enough. For example, [alpine configures the number of rotated logs as 6](https://git.alpinelinux.org/aports/tree/community/cloud-init/cloud-init.logrotate). Note that the compression algorithm / extension can also be configured.

To be safer here, I think we should glob and capture any file matching `f"{_log}*"`.

Reply:
Agreed and implemented